src/pkgcheck/checks/codingstyle.py

```diff
@@ -903,3 +903,102 @@ def feed(self, pkg):
                         yield RedundantDodir(
                             cmd.group('cmd'), line=dodir.group('call'),
                             lineno=lineno - 1, pkg=pkg)
+
+
+class EbuildUnquotedVariable(results.LineResult, results.Style):
+    """Variable is unquoted, but should be quoted"""
+
+    def __init__(self, variable, **kwargs):
+        super().__init__(**kwargs)
+        self.variable = variable
+
+    @property
+    def desc(self):
```

Comment:
After changing the result to handle multiple lines I'd make this function do something like:

```python
s = pluralism(self.lines)
lines = ', '.join(map(str, self.lines))
return f'unquoted variable {self.variable}, line{s} {lines}'
```

Reply:
done